### ISSUE TITLE
Use case: use CAM-KP to "explode" an edge

### ISSUE THREAD

**Issue by gaurav** [ExposuresProvider/cam-kp-api#537]
Given an edge, can CAM-KP provide a graph that replaces that edge with a series of edges and nodes that describes how that edge is defined in one or more pathways.

Benefits:
- Explains the relationship between the two nodes
- Provides additional nodes for follow-up queries

**Comment by gaurav** [comment]
Chris things "[X] upregulates/downregulates [geneX]" might be a templated query later this year, so that's what we should be focusing on.

**Comment by karafecho** [comment]
Here are two different ARAX queries that you can pull gene-chemical edges or just genes from, as described on slide 8 in [this deck](https://docs.google.com/presentation/d/1SjauAbONjnte4IHeCYIq3I7pS47Oc0faL3Uruh7nd4A/edit?usp=sharing):

https://arax.ncats.io/?r=44679 
https://arax.ncats.io/?r=52713

**Comment by karafecho** [comment]
I'm not sure I'm fully understanding your comment, but essentially, I'm seeking a way to invoke CAMs to better explain the chemical - gene edges in the workflow below. For instance, the current predicate linking biolink:Gene to biolink:ChemicalEntity (or biolink:Drug, Small Molecule) in Paths A and B is biolink:affects, which isn't terribly informative. Can we construct a TRAPI query that targets CAM and returns more informative information, e.g., biolink:ChemicalEntity - biolink:up/downregulates - biolink:Gene? The tissue/cell specificity is not an issue in Path A, but in Path B, you could use the output from the first hop to biolink:GrossAnatomicalStructure to anchor the results, if that makes sense.

![image](https://user-images.githubusercontent.com/26254388/195875229-84598216-8480-4c8c-b794-8873cc418091.png)

**Comment by gaurav** [comment]
Hi Kara! I think it's okay to wait until we get the other use case (#536) sorted out before we need to dive deeply into this. However, if you'd like to meet to go over what I've been working on, that would be fine! I'm hoping to have the lookup endpoint (#572) done and backend issues (https://github.com/ExposuresProvider/cam-pipeline/issues/80) fixed by early next week, so sometime next week might be a good time to meeting. Let me know!

**Comment by karafecho** [comment]
I agree that this can wait, now that I better understand the plan. Sure, let's meet after the lookup endpoint is active. Thanks!
